release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1300,6 +1300,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * Technology preview badges were displayed on the OpenShift Pipelines workflows in the web console, though OpenShift Pipelines was GA. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945153[*BZ#1945153*])
 
+* Previously, *Pipelines* you created in the git import flow for private repos failed to run. This happened because *Pipeline* ServiceAccount did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline ServiceAccount's annotations, and add pipeline-specific annotations to the provided secret. As a result, *Pipelines* for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
+
+* Previously, the *Start Modal* did not recognize an empty string as a valid entry, so you had to enter a value. This happened even though the *Tekton Operator* recognizes empty strings as a valid `parameter default`. With this update, you have the option to enter an empty string. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[BZ#1966275])
+
+* Previously, a decorator was shown for revisions in *Topology view* because annotations were passed to specifications of knative service along with metadata. Now the required decorators are shown for the knative service only in the *Topology view* and not with the associated revisions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954962[BZ#1954962])
```

Comment:
Knative - K is always capital in Knative

Reply:
Done